tests/cypress/support/commands.ts

```diff
@@ -177,7 +193,19 @@ Cypress.Commands.add('deleteApplicationDeployment', (clusterName='local') => {
   cypressLib.burgerMenuToggle();
   cypressLib.accesMenu(clusterName);
   cy.clickNavMenu(['Workloads', 'Deployments']);
-  // For certain reason deleteAll() is not working
-  // TODO: Investigate and fix it.
-  cy.deleteAllResources();
+  cy.wait(500);
+  cy.deleteAll(false);
+});
+
+// Modify given application
+Cypress.Commands.add('modifyDeployedApplication', (appName, clusterName='local') => {
```

Comment:
If not mistaken this function increases the count by 1
I guess to honor the name we could add logic to increase number to whatever is desired.
It can be done in the future.

Reply:
Discussed offline to have logic which I have commented as a comment.